src/base/bittorrent/torrentcreator.cpp

```diff
@@ -205,13 +206,29 @@ void TorrentCreator::run()
 
         checkInterruptionRequested();
 
-        // create the torrent
-        const nonstd::expected<void, QString> result = Utils::IO::saveToFile(m_params.savePath, entry);
+        const auto result = std::invoke([torrentFilePath = m_params.torrentFilePath, entry]() -> nonstd::expected<Path, QString>
```

Comment:
shouldn't you include `<functional>` header for `std::invoke`?

Reply:
All comments are addressed.